src/main/resources/templates/macros/report/output.vm

```diff
@@ -1,20 +1,49 @@
-#macro(includeOutput, $output)
+#macro(includeOutput, $outputs)
 
-#if ($output)
-  <div class="outputs inner-level">
-    #foreach($message in $output.getMessages())
+#set($combineOutputs = "true")
+#foreach($output in $step.getOutput())
+  #if(${output.getMessages().size()} > 1)
```

Comment:
move it to Java with friendly name eg hasTwoOrMoreMessages() - maybe you can find something shorter